src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/csip/CSIPPrisonOffenderEventListener.kt

```diff
@@ -71,6 +72,10 @@ class CSIPPrisonOffenderEventListener(
         RETRY_SYNCHRONISATION_MAPPING.name -> csipSynchronisationService.retryCreateCSIPReportMapping(
           sqsMessage.Message.fromJson(),
         )
+
+        RETRY_CSIP_FACTOR_SYNCHRONISATION_MAPPING -> csipFactorSynchronisationService.retryCreateCSIPFactorMapping(
```

Comment:
Ensure pick up retries

Reply:
Are you missing a test here than? Where mapping create fails once? I think that is the only "unhappyish" path worth doing